customizations.md

```diff
@@ -101,11 +101,11 @@ If all ECUs do have additional storage, and can perform a rollback, then the OEM
+In the first option, which we expect to be the common case, a fleet manager would configure the map file on ECUs such that Primaries and full verification Secondaries would trust an image if, and only if, it has been signed by both the OEM-managed Image repository, and the fleet-managed Director repository. Partial verification Secondaries would trust an image, if and only, if it has been signed by this Director repository. The upside of this option is that the fleet manager, instead of the OEM, has complete control over which updates are installed on its vehicles. The downside of this option is that should the fleet-managed Directory repository be compromised, attackers can execute mix-and-match attacks.
 
-In the second option, a fleet manager would configure the map file on ECUs such that Primaries and full verification Secondaries would trust an image if and only if it has been signed by three repositories: the OEM-managed Image repository, the OEM-managed Director repository, and the fleet-managed Director repository. The upside of this option is that attackers cannot execute mix-and-match attacks if they have compromised only the Director repository managed by either the OEM or the fleet. The downside of this option is that updates cannot be installed on vehicles unless both the OEM and fleet agree on which images should be installed together. This agreement may require both Director repositories to communicate using an out-of-band channel. Using this option also means that partial verification Secondaries should be configured to trust the Director repository managed by either the OEM or the fleet, but not both, since these Secondaries can only check for one signature.
+In the second option, a fleet manager would configure the map file on ECUs such that Primaries and full verification Secondaries would trust an image if, and only if, it has been signed by three repositories: the OEM-managed Image repository, the OEM-managed Director repository, and the fleet-managed Director repository. The upside of this option is that attackers cannot execute mix-and-match attacks if they have compromised only one of the Director repositories. The downside of this option is that updates cannot be installed on vehicles unless both the OEM and fleet agree on which images should be installed. This agreement may require both Director repositories to communicate using an out-of-band channel. Using this option also means that partial verification Secondaries should be configured to trust the Director repository managed by either the OEM or the fleet, but not both, since these Secondaries can only check for one signature.
```

Comment:
I would use the same alternate suggested text above.
```suggestion
In the second option, a fleet manager would configure the map file on ECUs such that Primaries and full verification Secondaries would *only* trust an image that has been signed by three repositories: the OEM-managed Image repository, the OEM-managed Director repository, and the fleet-managed Director repository. The upside of this option is that attackers cannot execute mix-and-match attacks if they have compromised only one of the Director repositories. The downside of this option is that updates cannot be installed on vehicles unless both the OEM and fleet agree on which images should be installed. This agreement may require both Director repositories to communicate using an out-of-band channel. Using this option also means that partial verification Secondaries should be configured to either trust the Director repository managed by the OEM or by the fleet, but not both, since these Secondaries can only check for one signature.
```

Reply:
This is fine, or as above, instead of emphasizing "only", emphasize "three".

Reply:
Ok. I'll be fixing that in a follow-up commit that fixes the wording of "map file"